repose-aggregator/tests/release-verification/src/vagrant/deb/Vagrantfile

```diff
@@ -0,0 +1,172 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+repose_version = ENV['REPOSE_VERSION'] || 'local'
+
+$update_yum = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Updating Apt packages"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+# update package lists from the server
+apt-get update -qq
+#apt-get upgrade -yq
```

Comment:
I figured that was commented out as some package upgrade wasn't playing nice. Also, since we don't declare a version on the base, we should always get relatively up-to-date packages from that.

Even so, we can totally un-comment this if no one has a reason not to.